classes/base.lua

```diff
@@ -139,16 +162,16 @@ function class.declareSettings (_)
   })
 end
 
-function class:loadPackage (packname, options)
+function class:loadPackage(packname, options)
```

Comment:
Greetings @jodros 
This PR touches several lines unrelated to your change (adding or removing spaces, changing indentation....) -- I think this should be avoided (or done separately when wanted, but I don't think it's the case here), notably;
- For `git blame` purpose when investigating issue
- For other PRs to be minimally impacted (e.g. my own #1853 touches upon the same areas) and avoid useless conflicts

Reply:
I also think the commit log is longer than expected, mentioning line numbers etc. - not so future-proof.

Reply:
Still applies see ex. below

Reply:
On the commits & merge vs rebase issue, for a small change like this I don't care too much since we can squash the whole thing to one `feat:` commit, but yes for larger change sets with any sort of incremental work done or large enough to merit separate testing and documentation commits having cleaner history helps PR review.